extension/e2e-tests/onboarding.test.ts

```diff
@@ -6,6 +6,7 @@ test.beforeEach(async ({ page, extensionId }) => {
 });
 
 test("Welcome page loads", async ({ page }) => {
+  await page.locator(".Welcome__column").waitFor();
```

Comment:
For some reason we need to wait for the page load here now, I think maybe related to the longer initial load now due to API calls in settings.